WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -452,7 +452,8 @@ nick_the_fair = {
 		NOT = { age = 50 }
 	}
 	chance = {
-		factor = 1
+		factor = 1.5
+
```

Comment:
That is not what you did though. This just makes this one more common at the expense of others.